examples/step-12/step-12.cc

```diff
@@ -644,6 +644,8 @@ int main()
 {
   try
     {
+      dealii::deallog.depth_console(5);
```

Comment:
Is that the right level? What does it actually show then?

Reply:
5 is big enough to show exactly what you see in https://www.dealii.org/developer/doxygen/deal.II/step_12.html#Results